src/test/java/LoginTests.java

```diff
@@ -3,23 +3,59 @@
 import org.openqa.selenium.chrome.ChromeOptions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import Pages.HomePage;
+import Pages.LoginPage;
 
 import java.time.Duration;
 
 public class LoginTests extends BaseTest {
     @Test
-    public void LoginEmptyEmailPasswordTest() {
+    public void LoginInvalidEmailValidPasswordTest() {
 
+        LoginPage loginPage = new LoginPage(getDriver());
+
+        loginPage.provideEmail("invalid@email.com").providePassword("te$t$tudent").clickSubmit();
+
+        Assert.assertEquals(getDriver().getCurrentUrl(), url); //"https://qa.koel.app/"
+    //
 //      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
+        //ChromeOptions options = new ChromeOptions();
+        //options.addArguments("--remote-allow-origins=*");
+
+        //WebDriver driver = new ChromeDriver(options);
+        //driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        //String url = "https://qa.koel.app/";
+        //driver.get(url);
+        //Assert.assertEquals(driver.getCurrentUrl(), url);
+        //driver.quit();
+    }
+    @Test
+    public void loginValidEmailEmptyPasswordTest() {
+        LoginPage loginPage = new LoginPage(getDriver());
 
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        loginPage.provideEmail("invalid@email.com").providePassword("te$t$tudent").clickSubmit();
 
-        String url = "https://qa.koel.app/";
-        driver.get(url);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-        driver.quit();
+        Assert.assertEquals(getDriver().getCurrentUrl(), url); //"https://qa.koel.app/"
     }
+    @Test
+    public void loginValidPasswordEmptyEmailTest() {
+        LoginPage loginPage = new LoginPage(getDriver());
+
+        loginPage.provideEmail("").providePassword("").clickSubmit();
+
+        Assert.assertEquals(getDriver().getCurrentUrl(), url); //"https://qa.koel.app/"
+    }
+   /* @Test
+    public void LoginValidEmailPassWordTest() {
+        LoginPage loginPage = new LoginPage(getDriver());
+
+        loginPage.provideEmail("trianasahlin@gmail.com").providePassword("te$t$tudent").clickSubmit();
+
+        String homePageLink = "https://qa.koel.app/#!/playlist/61648";
+
+        Assert.assertEquals(getDriver().getCurrentUrl(), homePageLink);
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.